src/App/Resources/AppResources.resx

```diff
@@ -2592,4 +2592,10 @@ Do you want to switch to this account?</value>
   <data name="OrganizationSsoIdentifierRequired" xml:space="preserve">
     <value>Organization SSO identifier required.</value>
   </data>
+  <data name="UpdateWeakMasterPasswordWarning" xml:space="preserve">
+    <value>Your master password does not meet one or more of your organization policies. In order to access the vault, you must update your master password now. Proceeding will log you out of your current session, requiring you to log back in. Active sessions on other devices may continue to remain active for up to one hour.</value>
+  </data>
+  <data name="CurrentMasterPass" xml:space="preserve">
+    <value>Current master password</value>
+  </data>
```

Comment:
Key and Value should be the same when the value's length is short => `CurrentMasterPassword` so it's easier to "guess" the exact value of the resource just by looking at the key